test/development/app-dir/next-after-app-invalid-usage/index.test.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable jest/no-standalone-expect */
+/* eslint-env jest */
+import { nextTestSetup } from 'e2e-utils'
+import * as Log from './utils/log'
+import {
+  assertHasRedbox,
+  getRedboxDescription,
+  getRedboxSource,
+} from '../../../lib/next-test-utils'
+
+describe('unstable_after() - invalid usages', () => {
+  const { next, isNextDev, skipped } = nextTestSetup({
+    files: __dirname,
+    // reading runtime logs is not supported in a deployed environment
+    skipDeployment: true,
+  })
+
+  if (skipped) return
```

Comment:
As far as I know this is not needed in `test/development`.

```suggestion
  })
```